x-pack/plugins/synthetics/public/apps/synthetics/components/monitors_page/overview/overview/monitor_detail_flyout.tsx

```diff
@@ -0,0 +1,460 @@
+                      values: [location],
+                    },
+                  ],
+                  dataType: 'synthetics',
+                  selectedMetricField: 'monitor.duration.us',
+                  name: 'All monitors response duration',
+                },
+                {
+                  seriesType: 'line',
+                  color: '#ddaf84',
+                  time: {
+                    from: previousDurationChartFrom ?? DEFAULT_PREVIOUS_DURATION_CHART_FROM,
+                    to: previousDurationChartTo ?? DEFAULT_PREVIOUS_DURATION_CHART_TO,
+                  },
+                  reportDefinitions: {
+                    'monitor.id': [id],
+                    'observer.geo.name': [location],
+                  },
+                  filters: [
+                    {
+                      field: 'observer.geo.name',
+                      values: [location],
+                    },
+                  ],
+                  dataType: 'synthetics',
+                  selectedMetricField: 'monitor.duration.us',
+                  name: 'Previous period',
+                },
+              ]}
+            />
+            <EuiSpacer />
+            <EuiTitle size="s">
+              <h3>{MONITOR_DETAILS_HEADER_TEXT}</h3>
+            </EuiTitle>
+            <EuiSpacer />
+            <BodyInfo
+              header={LAST_RUN_HEADER_TEXT}
+              content={
+                monitorDetail.data?.timestamp ? (
+                  <Time timestamp={monitorDetail.data?.timestamp} />
+                ) : (
+                  ''
+                )
+              }
+            />
+            <BodyInfo
+              header={LAST_MODIFIED_HEADER_TEXT}
+              content={
+                monitorSavedObject.updated_at ? (
+                  <Time timestamp={monitorSavedObject.updated_at} />
+                ) : (
+                  ''
+                )
+              }
+            />
+            <BodyInfo header={MONITOR_ID_ITEM_TEXT} content={props.id} />
+            <BodyInfo
+              header={MONITOR_TYPE_HEADER_TEXT}
+              content={capitalize(monitorSavedObject.type)}
+            />
+            <BodyInfo
+              header={FREQUENCY_HEADER_TEXT}
+              content={freqeuncyStr(monitorSavedObject?.attributes.schedule)}
+            />
+            <BodyInfo
+              header={TAGS_HEADER_TEXT}
+              content={
+                <>
+                  {monitorSavedObject?.attributes.tags?.map((tag) => (
+                    <EuiFlexItem key={`${tag}-tag`} grow={false}>
+                      <EuiBadge color="hollow">{tag}</EuiBadge>
+                    </EuiFlexItem>
+                  ))}
+                </>
+              }
+            />
+            <BodyInfo
+              header={URL_HEADER_TEXT}
+              content={
+                monitorDetail.data?.url?.full ? (
+                  <EuiLink href={monitorDetail.data.url.full}>
```

Comment:
Add `external` or `target="_blank"` to `EuiLink` so that the "popOut" icon appears -> this helps indicate to user that this will open a page in a new tab.

Reply:
9c8083c5925161e6703e28178304e3d707a2edd3